src/schoology-extractor/data_extractor/lib/__init__.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
About "lib" directory name, not this file.  Since it's a module name, would prefer something more descriptive.

Reply:
done